syntax/std_runtime.go

```diff
@@ -0,0 +1,52 @@
+package syntax
+
+import (
+	"strings"
+
+	"github.com/arr-ai/arrai/rel"
+)
+
+// BuildInfo represents arr.ai build information.
+var BuildInfo rel.Value
+
+func stdRuntime() rel.Attr {
+	return rel.NewTupleAttr("arrai",
+		rel.NewAttr("info", BuildInfo),
+	)
+}
+
+// GetBuildInfo returns arr.ai build information.
+func GetBuildInfo(version, date, fullCommit, tags, os, arch, goVersion string) rel.Value {
+	gitInfo := rel.NewTuple(
+		rel.NewAttr("commit", rel.NewString([]rune(fullCommit))),
+		// param tags has only one tag now.
+		rel.NewAttr("tags", rel.NewArray(rel.NewString([]rune(tags)))))
```

Comment:
Use a set, not an array.

Reply:
Set will output `tags: {  'v0.99.0' }`, and the expected output is `tags: ['v0.72.0']`.

Reply:
Resolved as https://github.com/arr-ai/arrai/pull/521#discussion_r460511677